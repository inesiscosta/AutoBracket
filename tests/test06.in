25 30
18 23 19 4 19 21 8 19 22 12 24 19 3 14 24 19 11 2 13 24 16 15 5 1 1
18 18 9 19 4 14 22 3 25 18 1 6 14 6 25 3 17 13 22 5 9 11 8 10 14
6 20 3 14 18 22 3 17 20 18 16 23 7 24 7 6 22 11 16 4 19 10 24 17 14
23 6 25 4 1 1 1 18 21 13 5 8 13 21 3 8 22 4 17 4 8 1 11 9 9
19 3 5 11 18 15 3 25 2 14 6 11 4 1 8 17 8 7 12 12 24 23 12 14 11
22 25 4 18 21 24 16 2 13 5 14 6 1 10 23 15 15 13 13 25 16 14 11 21 18
22 18 1 20 8 24 25 4 4 15 6 16 18 10 1 1 7 14 24 6 19 6 13 23 25
7 5 2 19 24 5 23 15 21 4 8 4 6 12 12 24 10 6 7 1 17 12 7 23 17
19 22 1 9 14 4 17 3 14 8 10 5 8 22 5 20 8 3 7 2 18 25 2 11 22
6 15 8 4 3 24 22 23 11 16 15 10 1 15 14 25 1 15 20 3 14 11 9 16 25
15 25 23 18 14 7 4 10 20 19 2 21 12 25 22 13 6 3 9 5 16 19 25 8 23
6 14 21 10 8 3 4 23 8 7 19 7 12 10 1 16 20 9 18 18 10 15 5 18 7
24 20 8 3 12 8 22 5 4 18 10 8 13 5 3 3 19 6 22 22 12 9 14 8 3
13 8 16 2 7 19 2 4 2 17 23 11 25 3 13 4 12 1 11 13 21 24 23 2 2
3 25 24 2 8 15 20 7 6 1 11 20 24 23 11 9 17 23 1 5 4 19 9 24 8
1 15 5 8 11 9 13 11 15 6 10 15 24 15 15 15 24 17 8 11 24 18 15 13 23
19 11 4 6 16 11 19 6 3 21 5 5 22 19 11 21 9 15 23 16 22 14 2 24 15
1 12 4 2 21 16 3 25 19 19 11 16 5 2 5 11 3 12 22 23 14 7 23 11 3
2 2 22 6 1 10 11 25 1 4 16 18 18 23 17 2 18 17 14 16 14 20 1 16 6
19 21 21 24 9 20 25 24 24 2 16 15 12 2 5 23 9 11 8 22 13 1 21 1 3
11 24 14 22 7 11 8 24 6 1 19 2 22 3 24 18 6 12 1 24 17 25 5 2 18
8 16 21 20 25 9 8 1 19 16 13 1 4 23 25 6 12 25 7 14 4 11 3 2 17
7 1 13 2 2 20 6 15 11 12 2 23 15 12 10 21 22 9 16 3 2 5 22 1 22
17 15 11 25 21 14 21 12 8 20 10 21 10 17 12 6 4 8 17 18 15 16 11 20 12
8 5 17 5 17 22 11 4 24 2 22 23 8 24 24 18 24 6 3 6 23 18 7 17 19
24 12 19 22 2 13 12 19 8 15 5 17 1 5 8 24 17 9 12 8 8 9 22 6 3 5 17 1 19 11
18
