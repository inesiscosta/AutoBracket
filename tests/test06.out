1
((((((((((((((((((((((24 ((((12 19) 22) 2) 13)) 12) 19) 8) 15) 5) 17) 1) 5) 8) 24) 17) 9) 12) 8) 8) 9) 22) 6) 3) (5 (17 1))) (19 11))
